packages/eslint-plugin/tests/rules/member-ordering.test.ts

```diff
@@ -5282,6 +5282,61 @@ class Foo {
           { default: { memberTypes: defaultOrder, order: 'alphabetically' } },
         ],
       },
+
+      // default option + class + defaultOrder + alphabetically
+      {
+        code: `
+class Foo {
+  public static a: string;
+  protected static b: string = "";
+  private static c: string = "";
+
+  public d: string = "";
+  protected e: string = "";
+  private f: string = "";
+
+  constructor() {}
+
+  get h() {}
+
+  set g() {}
+}
+            `,
+        options: [
+          {
+            default: {
+              memberTypes: defaultOrder,
+              order: 'alphabetically',
+            },
+          },
+        ],
+      },
+
+      // default option + class + custom + alphabetically
+      {
+        code: `
+class Foo {
+  @Bar
+  get a() {}
+
+  get b() {}
+
+  @Bar
+  set c() {}
+
+  set d() {}
+}
+            `,
+        options: [
+          {
+            default: {
+              memberTypes: ['get', 'decorated-get', 'set', 'decorated-set'],
+              order: 'alphabetically',
+            },
+          },
+        ],
+      },
```

Comment:
this isn't correct, right?
your config has specified that the ordering should be `get` and then `decorated-get`, but the code itself has a decorated get followed by a normal get.

Reply:
Thanks. Don't know how I overlooked that

Reply:
Fixed

Reply:
To clarify - the problem was less that the test was wrong and yet it was passing fine.

Which means you haven't changed the rule correctly to handle the feature.

Reply:
If you check the code the actual test was wrong. Or at least its placement because it is in the valid section.
And checking my PR you can see updated code.

P.S. As soon as I updated the code the test started to fail which proves it right. So it was updated as well. Again because it's in the valid section.

Reply:
The bottom line is that now it's working correctly and if I were to leave the test unchanged I'd have to move to the invalid section and add the expected errors so to have it passing.

Reply:
Just added another test exactly like it was when you commented. This time to the invalid section to prove the code works correct